doc/ingest.md

```diff
@@ -86,6 +88,32 @@ The network indexer nodes expect that metadata begins with a `uvarint` identifyi
   * the proposed `uvarint` protocol is `0x3D0000`.
   * the following bytes are not yet defined.
 
+#### ExtendedMetadata
+
+The `ExtendedMetadata` field allows for specification of provider families, in cases where a provider operates multiple PeerIDs, perhaps with different access methods between them, but over the same database of content.
+
+```
+type ExtendedMetadata struct {
+    Family [ExtendedProvider]
+    Override bool
+}
+
+type ExtendedProvider struct {
+    Provider String
+    Addresses [String]
+    Metadata optional Bytes
+    Signature Bytes
+}
+```
+
+* If Metadata is not specified for an `ExtendedProvider`, the metadata for an individual `Advertisement` will be used instead.
+* If an `ExtendedProvider` listing is written with no `ContextID`, those peers will be returned for all published advertisements for the advertisement Publisher.
+  * If `Override` is set on an `ExtendedProvider` entry on an advertisement with a `ContextID`, that indicates that any set chain-level set of providers should not be returned for that context ID.
+  * If `Override` is not set on an entry for an advertisement with a `ContextID`, it will extended any set chain-level `ExtndedProviders`.
+* The `Signature` for an `ExtendedProvider` is made using the key of that extended `Provider`.
+  * The full advertisement object is serialized, with all instances of `Signature` replaces with an empty array of bytes.
```

Comment:
It would - but with the signature sections in it removed - that does force choosing the full set of identities in advance of the signing, but prevents subsequent partial sub-identity addition/removals

Reply:
Do we not want to allow "partial sub-identity addition/removals" given that in the estuary usecase this new identities can come and go dynamically?

Reply:
by this do you mean just specifying a delta (eg "the identity d is no longer providing") as opposed to writing out the current family each time there is a delta as would be specified currently (eg "the current family is now a,b, and c" no longer including d)

Reply:
It'd be good to allow publishers to add / remove extended providers without having to re-obtain a signature from each of them for each change. I.e. given that a provider has confirmed (signed an AD) that they can provide a certain piece of data, they don't necessarily need to confirm it again once another party can provide the same piece of data too. 

So maybe extended providers could sign only over the AD content, without `ExtendedMetadata` included?

Reply:
the extended provider needs to sign over it's own multi-addrs to make sure that it isn't being mis-represented

Reply:
Maybe a provider could sign the AD without `ExtendedMetadata` but with *their* details filled in as if they were a primary provider? Or signing with `ExtendedMetadata` included only for *that* provider could work too. Wdyt?

Reply:
see if you think the current setup is reasonable